### PR TITLE
I made my homework week1

### PR DESCRIPTION
**Pull request by Kani103**

### CODE REVIEW

File: Week1/database.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    database: 'userdb'
+});
+
+let database = 'create database IF NOT EXISTS new_world';
+let useDatabase = 'use new_world';
+
+let tableCountry = ` CREATE TABLE IF NOT EXISTS country(
+    name CHAR(20),
+    population INT,
+    ends_land CHAR(20),
```
Comment (unmeshvrije) [inline Week1/database.js]: There is no country_code field here.
Please check all queries carefully before you design the table.

File: Week1/database.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      `select * from country where continent='Europe';`,
+      `select * from country order by surfaceArea DESC;`,
+      `select * from city where countrycode = 'NLD';`,
+      `select population from city where name = 'Rotterdam';`,
+      `select name from country order by surfaceArea desc limit 10;`,
+      `select name from city order by population DESC limit 10;`,
+      `select sum(population) from country;`,
+];
+
+let insertQueries = [
```
Comment (unmeshvrije) [inline Week1/database.js]: Are these insert queries ? 
Looks like console messages to me.

File: Week1/database.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+let mysql = require('mysql');
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'userdb'
```
Comment (unmeshvrije) [inline Week1/database.js]: Using database 'userdb' is not necessary if you are going to create new_world database.

**Comment by unmeshvrije** [comment]
reviewed.

**Comment by unmeshvrije** [comment]
reopening for changes.

**Comment by wouterkleijn** [comment]
thanks @unmeshvrije @Kani103 Still waiting for your changes

**Comment by Kani103** [comment]
Hey Unmesh, sorry for the delay..
the answers are working well and i added a screenshots folder too.
the insert queries not working properly i tried to fix them but no success.
thanks in advance..

**Pull request closed without merging**